### ISSUE TITLE
Where to store the funds?

### ISSUE THREAD

**Issue by reiven** [coinspark/python-OP_RETURN#4]
Hi
I'm trying to use this lib, and understand where should the funds needed to use the lib should be store, and as fas as i can understand, the lib uses the bitcoin core commands using rpc, so i assume i need the funds in the wallet of the bitcoin core client, thats correct?
Maybe it will be useful to add this information into the readme

**Comment by gidgreen** [comment]
Yes, in the node's wallet. I thought that was obvious but maybe not!

**Comment by reiven** [comment]
No, i guess it will be a good idea to note that in the Readme file.
Also: there is any way to specify which 'account' in the bitcoin wallet to use?

**Comment by reiven** [comment]
Oh, also i'm getting an http error 500 and doing the request by hand i get this message from the wallet: "Please enter the wallet passphrase with walletpassphrase first.".
There is no setting for that, correct?

**Comment by gidgreen** [comment]
Right, the library does not yet support encrypted wallets.
